packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -553,6 +608,58 @@ test.serial('multiple Depositors, not all assets are sold', async t => {
   await assertPayouts(t, liqSeatB, currency, collateral, 385n, 166n);
 });
 
+// serial because dynamicConfig is shared across tests
+test.serial('multiple Depositors, with goal', async t => {
+  const { collateral, currency } = t.context;
+  const driver = await makeAuctionDriver(t);
+
+  const liqSeatA = await driver.setupCollateralAuction(
+    collateral,
+    collateral.make(1000n),
+  );
+  const liqSeatB = await driver.depositCollateral(
+    collateral.make(500n),
+    collateral,
+    { goal: currency.make(300n) },
+  );
+  await driver.updatePriceAuthority(
+    makeRatioFromAmounts(currency.make(11n), collateral.make(10n)),
+  );
+
+  const result = await E(liqSeatA).getOfferResult();
+  t.is(result, 'deposited');
+
+  await driver.advanceTo(167n);
+  const seat = await driver.bidForCollateralSeat(
+    currency.make(1500n),
+    collateral.make(1000n),
+  );
+
+  t.is(await E(seat).getOfferResult(), 'Your bid has been accepted');
+  t.false(await E(seat).hasExited());
+
+  await driver.advanceTo(170n);
+  await eventLoopIteration();
```

Comment:
I forget, if we make `advanceTo` also await the event loop something else breaks?

WDYT of having an option like…
```suggestion
  await driver.advanceTo(170n, 'andTurn');
```

incidental, out of scope